data/fajnzylber2020sars/fajnzylber2020sars.yaml

```diff
@@ -0,0 +1,1733 @@
+# yaml-language-server: $schema=../.schema.yaml
+title: SARS-CoV-2 viral load is associated with increased disease severity and mortality
+doi: 10.1038/s41467-020-19057-5
+description: >
+  The paper quantified SARS-CoV-2 viral load from participants with a diverse range
+  of COVID-19 disease severity, including those requiring hospitalization, outpatients
+  with mild disease, and individuals with resolved infections. Nasopharyngeal swabs,
+  oropharyngeal swabs, sputum, and urine were collected from hospitalized participants.
+  Data were obtained from the supplementary materials.
+analytes:
+  Nasopharyngeal_SARSCoV2_N:
+    description: >
+      Nasopharyngeal swabs were collected in 3 mL of phosphate-buffered saline. Viral
+      concentrations were quantified using RT-qPCR targeting the N gene in the nasopharyngeal
+      swab samples.
+    limit_of_quantification: unknown
+    limit_of_detection: 40
+    specimen: nasopharyngeal_swab
+    biomarker: SARS-CoV-2
+    gene_target: N
+    unit: gc/mL
+    reference_event: symptom onset
+  Oropharyngeal_PBS_SARSCoV2_N:
+    description: >
+      Oropharyngeal swabs were collected in 3 mL of phosphate buffered saline. Viral
+      concentrations were quantified using RT-qPCR targeting the N gene in oropharyngeal
+      swab samples.
+    limit_of_quantification: unknown
+    limit_of_detection: 40
+    specimen: oropharyngeal_swab
+    biomarker: SARS-CoV-2
+    gene_target: N
+    unit: gc/mL
+    reference_event: symptom onset
+  Oropharyngeal_VTM_SARSCoV2_N:
+    description: >
+      Oropharyngeal swabs were collected in viral transport medium. Viral concentrations
```

Comment:
Do we know the volume of the viral transport medium? Is that also 3 mL?

Reply:
The methodology section mentioned that 'Nasopharyngeal swabs and oropharyngeal swabs were collected in 3 mL of phosphate-buffered saline,' but it did not specify the volume of the viral transport medium. Therefore, I did not include that information in the analyte description.